### PR TITLE
Hcf 1033 respect from option

### PR DESCRIPTION
**Pull request by ericpromislow**
4 types of commits here. In order of importance:

1. Code: fissile show layer no longer supports the `--from|-F`

2. Doc point 1.

3. Docs: change`/Users/vladi` in generated docs to `~`

4. Add a Makefile entry to regenerate markdown docs. Maintain the '~' for paths. Also throw away changes that change only the datestamps.

3. Doc the general `--output ` option in all the docs/*.md files

### CODE REVIEW

File: make/generate-markdown [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# Change expanded home directory back to '~'
+for x in $(git status | awk 'modified:\s+docs/ {print $2}') ; do
+    sed -i 's@'${HOME}'@~@g' $x
+done
+# Then check for date changes only, and revert if that's all we have.
+for x in $(git status | awk 'modified:\s+docs/ {print $2}') ; do
+    numUnmatched=$(git diff $x | grep -v -E '^[-+]###### Auto generated by spf13/cobra on [0-9]+-[a-zA-Z]+-[0-9]{4}' | wc -l)
+    case $numUnmatched in
+	0) git checkout -- $x ;;
+	*) #keep diffs in $x ;;
```
Comment (alexjh) [inline make/generate-markdown]: The `;;` needs to be before the comment

File: make/generate-markdown [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+GIT_ROOT=${GIT_ROOT:-$(git rev-parse --show-toplevel)}
+# Force an error if $HOME isn't set
+home=${HOME:?"HOME isn't set"}
+
+set -o nounset
+
+cd ${GIT_ROOT}
+./fissile docs markdown
+# Change expanded home directory back to '~'
+for x in $(git status | awk 'modified:\s+docs/ {print $2}') ; do
```
Comment (alexjh) [inline make/generate-markdown]: awk syntax error. Missing first `/` before `'modified...` ?

File: make/generate-markdown [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    sed -i 's@'${HOME}'@~@g' $x
+done
+# Then check for date changes only, and revert if that's all we have.
+for x in $(git status | awk 'modified:\s+docs/ {print $2}') ; do
```
Comment (alexjh) [inline make/generate-markdown]: Same syntax error.

File: make/generate-markdown [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    numUnmatched=$(git diff $x | grep -v -E '^[-+]###### Auto generated by spf13/cobra on [0-9]+-[a-zA-Z]+-[0-9]{4}' | wc -l)
```
Comment (alexjh) [inline make/generate-markdown]: You can save running `wc` with `grep -c`

File: make/generate-markdown (hunk 3 above)
Comment (ericpromislow) [inline make/generate-markdown]: Fixed in d7dc0ed

File: make/generate-markdown (hunk 2 above)
Comment (ericpromislow) [inline make/generate-markdown]: Fixed in d7dc0ed

File: make/generate-markdown (hunk 1 above)
Comment (ericpromislow) [inline make/generate-markdown]: Fixed in d7dc0ed (by putting the ';;' on a separate line, so the code is slightly more maintainable.

**Pull request merged**